Wording update (#3488)
DataDog/documentation: content/logs/_index.md

### Datadog Logs Endpoints

> Two endpoints are available to send logs to Datadog: one for SSL encrypted connections and another one for unencrypted connections.

Reviewer: ```suggestion
Datadog provides logging endpoints for both SSL-encrypted connections and unencrypted connections.
```

> You should use the SSL encryption when possible and the Datadog Agent uses an SSL encrypted connection to send logs to Datadog (more information available in the [Datadog security documentation][40]).

Reviewer: ```suggestion
You should use the encrypted endpoint when possible. The Datadog Agent uses the encrypted endpoint to send logs to Datadog (more information available in the [Datadog security documentation][40]).
```
